platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigration.java

```diff
@@ -164,30 +168,35 @@ public void export(@NotNull Path exportDirectory) throws MigrationException, IOE
             Configuration[] configurations = configurationAdmin.listConfigurations(FILTER);
             if (configurations != null) {
                 for (Configuration configuration : configurations) {
-                    Path exportedFilePath = etcDirectory
-                            .resolve(configuration.getPid() + configurationFileExtension);
+                    Path exportedFilePath = etcDirectory.resolve(
+                            configuration.getPid() + configurationFileExtension);
                     try {
-                        configurationFileFactory
-                                .createConfigurationFile(configuration.getProperties())
+                        configurationFileFactory.createConfigurationFile(configuration.getProperties())
                                 .exportConfig(exportedFilePath.toString());
                     } catch (ConfigurationFileException e) {
                         LOGGER.error("Could not create configuration file {} for configuration {}.",
-                                exportedFilePath, configuration.getPid());
-                        throw new MigrationException("Failed to export configurations.", e);
+                                exportedFilePath,
+                                configuration.getPid());
+                        throw new MigrationException(String.format(
+                                "Failed to export configurations: %s",
```

Comment:
Same as in `MigrationUtil`.  It looks like we're repeating `"Failed to export configurations:"` a lot.